lib/group.js

```diff
@@ -15,9 +15,9 @@ export function getGroupEntities(entities, group) {
 }
 
 // Split a collection into a list of groups and a 'rest' list of ungrouped
-// entities. Groups will be ordered by their order.
+// entities. Groups will be ordered by their order if 'sort' is true.
 // Returns { groups: [], ungrouped: {} }
-export function splitByGroups(entities) {
+export function splitByGroups(entities, sort = true) {
```

Comment:
Let's keep this function simple and remove sorting the groups array completely.

Where it is needed, the ordering can be done by the code calling this.

Make sure to update the test too.